src/components/network/selection-creation-panel.tsx

```diff
@@ -159,41 +185,66 @@ const FilterCreationPanel: React.FC<FilterCreationPanelProps> = ({
                     <GridSection title="createNewFilter" />
                     <Grid container paddingTop={2}>
                         <SelectInput
-                            name={'equipmentType'}
-                            options={equipmentTypesOptions}
-                            label={'EquipmentType'}
+                            name={SELECTION_TYPE}
+                            options={Object.values(SELECTION_TYPES).map(
+                                (value) => {
+                                    return {
+                                        id: value,
+                                        label: selectionTypeToLabel(value),
+                                    };
+                                }
+                            )}
+                            label={SELECTION_TYPE}
                             fullWidth
                             size={'medium'}
                             disableClearable={true}
                             formProps={{ style: { fontStyle: 'italic' } }}
                         />
                     </Grid>
+                    {watchSelectionType !== '' && (
+                        <>
+                            <Grid container paddingTop={2}>
```

Comment:
instead of repeating "paddingTop" each time, use "rowGap" for parent Grid

Reply:
done